packages/expo-font/src/Font.ts

```diff
@@ -12,15 +12,16 @@ import {
 const loaded: { [name: string]: boolean } = {};
 const loadPromises: { [name: string]: Promise<void> } = {};
 
+// @needsAudit
 /**
  * Used to transform font family names to the scoped name. This does not need to
  * be called in standalone or bare apps but it will return unscoped font family
  * names if it is called in those contexts.
  * note(brentvatne): at some point we may want to warn if this is called
  * outside of a managed app.
```

Comment:
Perhaps we don't want this to be included in the docs 😂

Reply:
In the latest commit I have moved the note outside of the method description, so it should be no longer visible in the docs.